matrix-appservice-irc/Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG bridge_version=latest
+FROM matrixdotorg/matrix-appservice-irc:${bridge_version}
+
+COPY config.yaml /data/config.yaml
+COPY irc-registration.yaml /data/appservice-registration-irc.yaml
+COPY tempkey.pem /data/passkey.pem
```

Comment:
instead of building this into the container, put it in the puppet private share and mount it in as a volume a la https://github.com/ocf/ocfweb/blob/d7d22d7ee88ce0599f69ea3d93f7de7749fd814f/kubernetes/ocfweb-web.yaml.erb#L38-L40 https://github.com/ocf/ocfweb/blob/d7d22d7ee88ce0599f69ea3d93f7de7749fd814f/kubernetes/ocfweb-web.yaml.erb#L70-L73. ref for how this works: https://github.com/ocf/puppet/blob/master/modules/ocf_kubernetes/manifests/worker/secrets.pp

Reply:
It's possible we don't need to our own Dockerfile here at all. I think it should be possible to mount these files into the container as volumes (which would be done in the Kubernetes pod config)

Reply:
That's completely fair in this situation, I'll look into removing this and mounting it.

I will probably keep a copy of config.yaml and an old copy of irc-registration.yaml around for example and move these all to the puppet share.